_posts/blog/2019-03-14/2019-03-14-how-to-sell-digital-goods-for-cryptocurrency-on-openbazaar.markdown

```diff
@@ -0,0 +1,69 @@
+
+This may blow your mind so proceed with caution.
+
+Where do you buy your ebooks? Okay, maybe still Amazon or similar...What about music and movies? Oh, now we’re starting to get a little weird. What about online courses? Digital access codes? Video game codes? Data bundles like fonts or photo editing presets? 
+
+Online marketplaces aren't all about physical goods. Digital goods are just as sellable as physical goods and there are many places that many people buy and sell them around the web.
+
+On OpenBazaar you can easily sell digital goods for cryptocurrency. Here’s how.
+
+#### 1. File hosting
+
+**File hosting** and **automatic delivery** within OpenBazaar is a feature we plan to build in the future but it is still on ice at the moment. In the meantime, if you are selling things like game codes or gift cards, you will have to send them manually to your buyers when you receive a payment. If you are selling a file that users will need to download, you will need to host it somewhere. 
+
+We would love to recommend a decentralized file hosting solution but the most promising projects are still in an early release stage or pre-release. If you’d like to learn more about projects like [Filecoin](https://filecoin.io/), [Storj](https://storj.io/) and [Sia](https://sia.tech/) you can do so [here](https://coingeek.com/p2p-decentralized-file-storage-dropbox-killer/) and [here](https://medium.com/bitfwd/what-is-decentralised-storage-ipfs-filecoin-sia-storj-swarm-5509e476995f).
+
+You can host files somewhere like your own external website or server, or by using a storage service like Dropbox or Google Drive. Dropbox Professional and Business plans are paid services but they allow you to set a password on files for an extra layer of security around who can access it.
+
+When you receive a payment you can then share a link with your buyer that they can click to download their file. Or, if you want to keep it very simple and your file is small enough you can let buyers know that they need to provide an email address so you can email the file to them when their payment is confirmed.
+
+#### 2. Create your listing
+
+Now that you have a plan for hosting and sharing your digital product you can create a listing for it on OpenBazaar. Create a **new listing** and under **Type** select **“Digital Good.”**
+
+Give it a great **title** and **descriptio**n that relates to keywords people may use in search to find it easily. Make sure to also describe how you will send the file to users when they complete their purchase so they know what to expect and where, whether in OpenBazaar chat or via email. It’s also helpful to include any instructions they may need for setting up or installing your file if it requires that.
+
+Include at least 1 great **image** that describes your item such as a book jacket mockup for an ebook or the cover art for a music album. 
+
+Leave the **shipping** option blank because you don’t need to worry about shipping! Hooray!
```

Comment:
What shipping option? If it's a digital listing there isn't a shipping section.

Reply:
Oh, thanks!